flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/stream/compact/CompactOperator.java

```diff
@@ -119,10 +117,11 @@ public void initializeState(StateInitializationContext context) throws Exception
     }
 
     @Override
-    public void processElement(StreamRecord<CoordinatorOutput> element) throws Exception {
-        CoordinatorOutput value = element.getValue();
-        if (value instanceof CompactionUnit) {
-            CompactionUnit unit = (CompactionUnit) value;
+    public void processElement(StreamRecord<CompactMessages.CoordinatorOutput> element)
+            throws Exception {
+        CompactMessages.CoordinatorOutput value = element.getValue();
```

Comment:
nit: you could also add a static import here and at other locations (e.g. `CompactCoordinator`) to keep the changes in implementation minimal. this would help keeping the git blame helpful.